nipype/interfaces/mrtrix3/utils.py

```diff
@@ -19,6 +18,43 @@
 from .base import MRTrix3BaseInputSpec, MRTrix3Base
 
 
+class TckSift2InputSpec(CommandLineInputSpec):
+    in_file = File(exists=True, argstr="%s", mandatory=True, position=-3, desc="input streamlines file")
+    in_fod = File(exists=True, argstr="%s", mandatory=True, position=-2, desc="input FOD file")
+    out_weights = File(argstr="%s", mandatory=True, position=-1, desc="output weights file")
+
+
+class TckSift2OutputSpec(TraitedSpec):
+    out_weights = File(exists=True, desc="output weights file")
+
+
+class TckSift2(CommandLine):
+    """
+    Interface for MRTrix's tcksift2 command
+
+    Example
+    -------
+
+    >>> import nipype.interfaces.mrtrix3 as mrt
```

Comment:
For this to work, this needs to be added to 

https://github.com/nipy/nipype/blob/a49f154b21a09f7ce19e5d48c4eada3506343f7d/nipype/interfaces/mrtrix3/__init__.py#L17-L36